### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by JauJau088**
In this Pull Request, I have created my first capstone project.

For more info about this project, please take a look at my [video here](https://www.loom.com/share/4c3416df28d74c36a2b89170c8eae818).

A code review is very much appreciated.

Cheers :)

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (MeqdamAlqudah) [inline .eslintrc.json:1]: - [ ] Please record a video presenting the features of the project:-

- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. - - - The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @JauJau088 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .eslintrc.json (hunk 1 above)
Comment (JauJau088) [reply .eslintrc.json:1]: Thank you. Updated.

**Review by vic778 — APPROVED** [review]
Hi @JauJau088, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**